modules/va_facilities/README.v1.yml

```diff
@@ -448,31 +449,49 @@ paths:
           in: query
           description: |
             Street address of the location from which drive time will be calculated.
-          required: true
+          required: false
```

Comment:
Why are these changing to false? Aren't they required?

Reply:
I was looking at v0 facilities openapi spec and it seemed to me that when having an either or situation that all the required params for each group had the required removed and was just documented in the endpoint description so I followed suit.

The other alternative is to have `street_address`, `city`, `state`, `zip` and `lat` and `lng` all required.  Not sure which option is more confusing

Reply:
Ah, got it.